tools/tf2openapi/README.md

```diff
@@ -0,0 +1,21 @@
+# Tf2OpenAPI
+The tool enables generating OpenAPI 3.0 specifications for prediction HTTP requests from TensorFlow SavedModel files containing signature definitions (SignatureDefs) for models. One use case for the OpenAPI specification involves generating sample payloads in a UI.
```

Comment:
This is a run on sentence :). Happy to help you clean up the language, or you can take another pass yourself.